lib/ui/homepage.dart

```diff
@@ -36,20 +35,22 @@ class _HomePageState extends State<HomePage> {
   }
 
   Widget _buildButton(child, onpress) {
-    var color = Theme.Colors.backgroundColor;
-    return FlatButton(
-      onPressed: onpress,
-      color: color,
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(19.0),
-        side: BorderSide(color: color),
-      ),
-      child: Container(
-        height: 50,
+    var color = theme.Colors.backgroundColor;
+    return Container(

```

Comment:
This is not the same as before, potentially giving a smaller button than the sourrnding container. Are you sure about this?

Reply:
Pretty sure the button size is the same (that's defined by the FlatButton and inner Container), this is just adding padding/margin around each button so that when laid out they don't rub against each other.

This comes in more in the vertical flow case because there's no set "height" to the overall container, so it will just pick a height of exactly 3 buttons high with no padding between them.

At least that's my understanding, this does look good both in portrait and landscape FWIW.